src/pages/documentation/faq.md

```diff
@@ -84,20 +84,37 @@ iOS
 
 ### What is the size of the SDK?
 
-| Extension | iOS (KB) | Android (KB) |
-| :--- | :--- | :--- |
-| Core | 504 | 168 |
-| Adobe Analytics | 54 | 21 |
-| Adobe Audience Manager | 40 | 13 |
-| Adobe Target | 77 | 27 |
-| Profile | 20 | 8 |
-| Adobe Campaign Standard | 60 | 30 |
-| Places | 36 | 20 |
-
-The size values in the table are provided as indicative estimates, with the following considerations:
-
-* Mobile Core, which includes the Lifecycle, the Identity, and the Signals extensions, is required for all other extensions. The final app size increase can be calculated by adding the Mobile Core size to each of the enabled extensions. For example, the iOS app distribution using the Target and Analytics extensions will have a total size increase of 635 KB. (Core: 504 KB + Analytics: 54 KB + Target: 77 KB).
-* The iOS (SDK extension versions 2+) estimates are based on Xcode’s App Thinning size report for one architecture. The Android (SDK extension versions 1+) size estimates listed refer to unsigned apps and do not account for proguarding.
+#### Adobe Experience Platform SDKs
+
+| Extension                 | iOS (KB) | Android (KB) | Additional Dependencies†                          |
+| :------------------------ | :------- | :----------- | :------------------------------------------------ |
+| Core                      | 925      | 497          | Android only: `kotlin-stdlib`                     |
+| Identity                  | 136      | 51           |                                                   |
+| Lifecycle                 | 150      | 59           |                                                   |
+| Signal                    | 18       | 33           |                                                   |
+| Edge Network extension    | 253      | 99           | Identity for Edge Network                         |
+| Consent for Edge Network  | 33       | 25           | Edge Network extension, Identity for Edge Network |
```

Comment:
I suggest we only list the direct dependencies, in this case Edge Network extension (which if you check the table above depends on Identity for Edge Network). - Same comment for the other extensions below.